tools/internal_ci/linux/grpc_build_submodule_at_head.sh

```diff
@@ -22,20 +22,24 @@ cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
+# Submodule name is passed as the RUN_TESTS_FLAGS variable
+SUBMODULE_NAME="${RUN_TESTS_FLAGS}"
+
 # Update submodule to be tested at HEAD
-(cd third_party/$RUN_TESTS_FLAGS && git fetch --all && git checkout origin/master)
+(cd "third_party/${SUBMODULE_NAME}" && git fetch origin && git checkout origin/master)
```

Comment:
What's the difference between 'git fetch origin' and 'git fetch --all' in this context?

Reply:
Fetch all fetches all the remotes, which kinda makes sense, but we only care about "origin" (that will always exist for submodules) and we're calling out origin/master in the next command anyway, so I thought let's be consistent (and improve readability).